lib/worker-pipeline.js

```diff
@@ -57,7 +58,15 @@ class AssetComputeWorkerPipeline {
             return this.transformers;
         }
 
-        const transformer = new WorkerCallbackTransformer(renditionCallback, this.options.manifest);
+        //const transformer = new WorkerCallbackTransformer(renditionCallback, this.options.manifest);
+        let transformer;
+        if (this.options.manifest.hasBatchMode || this.options.manifest.inputs.hasBatchMode) {
```

Comment:
remove the second option: `this.options.manifest.inputs.hasBatchMode`

Reply:
When we will have a stable API, yes, sure.